pages/fleet/[id].vue

```diff
@@ -66,6 +66,16 @@ import ProductIcon3 from '~/assets/icons/product_icon_3.svg?skipsvgo';
 const id = useRoute().params.id || '';
 const { data: zeppelin, isLoading } = useProductById(id);
 const { data: zeppelins, isLoading: isLoadingPopular } = useProductList(ref(4));
+const { show: showAuthModal } = useAuthModal();
+const { data: userProfile } = useUserProfile();
+
+const handleBook = () => {
+  if (userProfile.value) {
+    navigateTo(`/checkout/${zeppelin.value?.id}`);
+  } else {
+    showAuthModal();
+  }
+};
```

Comment:
The redirection after successful login is missing:
![image](https://user-images.githubusercontent.com/66744401/229552944-96416311-ff8e-44f6-ac8b-2f9684cd6065.png)

Check `AuthModal` component, there is a line of:
```ts
login({ email, password }, { onSuccess: hide });
```

Maybe we should change it so after successful login not only `hide()` method runs but if we clicked the Book button, it should run `hide()` and conditionally `navigateTo()` afterwards. Or if you have another idea, I'm open to it 😄